app/processing/transform-payment-request.js

```diff
@@ -1,12 +1,18 @@
-const confirmDueDates = require('./confirm-due-dates')
+const { BPS } = require('../constants/schemes')
+const { confirmPaymentRequestNumber } = require('./confirm-payment-request-number')
+const getCompletedPaymentRequests = require('./get-completed-payment-requests')
 const calculateDelta = require('./delta')
+const confirmDueDates = require('./confirm-due-dates')
 const enrichPaymentRequests = require('./enrichment')
-const getCompletedPaymentRequests = require('./get-completed-payment-requests')
 
 const transformPaymentRequest = async (paymentRequest) => {
+  // If BPS, then need to confirm payment request number as rekeyed claims can result in duplicate payment request numbers
```

Comment:
Normally I would say to remove the comments, but they've sort of been helping me while working on another ticket in this repo. So save removing them here and instead I'll remove them in my PR when I'm done with them, provided we don't actually need them - if that works?

Reply:
I think in this case we should keep them as we're doing some unexpected things to deal with side effects from Siti.      So I think this will be a helpful explanation to whoever supports this in future, if not us.

For comments I'd say we only add them if we're doing something weird so we can explain why, or we're doing something which is difficult to explain by reading the code.  Hopefully we don't have much if any of the latter

Reply:
Yeah, okay. I'm happy enough to keep them in them